snmp/tests/test_e2e_core_metadata.py

```diff
@@ -1,6 +1,8 @@
 # (C) Datadog, Inc. 2021-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
+import pprint
```

Comment:
@FlorianVeaux No, there is a real `pprint.pprint(event1['devices'])` line below. 

It will be only visible when test fail, it's mainly for convenience to have the pretty print of the devices when the test fail.